docs/introduction/best-practices.md

```diff
@@ -248,7 +250,8 @@ AFRAME.registerComponent('foo', {
 ```
 
 Again be careful what you do in tick functions, treat them as critical
-performance code because they will be run 90 times per second.
+performance code because they will be run 90 times per second. [Consider
+using `utils.throttle`][throttle] to run your code at less frequent intervals.
```

Comment:
Thanks! I think more common is throttleTick to wrap a tick function and throttle it.